test/functional/fixtures/regression/gh-973/pages/container.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>GH-913 container</title>
+</head>
+<body>
+<div style="height: 300px; width: 300px; position:absolute; left: 500px; top: 500px; background:yellow; overflow: scroll">
+    <div id='targetUpperLeft'
+         style="left:200px; top:200px; position:absolute; background: #150;    width: 50px;    height: 50px">
+        Upper left target
+    </div>
+    <div id='targetLowerRight'
+         style="left:2000px; top:2000px; position:absolute; background: #015;    width: 500px;    height: 500px">
+        Lower right target
+    </div>
+    <div
+        style="left:2700px; top:2700px; position:absolute; background: #000;    width: 50px;    height: 50px">
```

Comment:
it's better to use classes instead of this long style string. Keep only `left, top` inline (and maybe background)
```css
div {position: absolute}
.large {width: 500px; height: 500px}
.small ...
```